### PR TITLE
Fix npm audit warnings

### PR DESCRIPTION
**Pull request by jasonnutter**
Update/replace/remove packages that are failing npm audit checks. Resolved all warnings for core, angularjs, browser, common, node, and node extensions. Will do Angular for v2.

Results:
image

### CODE REVIEW

File: extensions/msal-node-extensions/package.json [hunk 1]
```diff
@@ -37,15 +37,15 @@
   },
   "module": "dist/test.esm.js",
   "dependencies": {
-    "bindings": "^1.5.0",
     "@azure/msal-common": "^1.6.3",
-    "keytar": "6.0.1",
+    "bindings": "^1.5.0",
+    "keytar": "^7.0.0",
```
Comment (jasonnutter) [inline extensions/msal-node-extensions/package.json:42]: @sameerag This shouldn't break anything, but worth testing: https://github.com/atom/node-keytar/releases/tag/v7.0.0

File: lib/msal-browser/package.json [hunk 2]
```diff
@@ -84,8 +84,8 @@
     "rollup": "^1.24.0",
     "rollup-plugin-json": "^4.0.0",
     "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-typescript2": "^0.24.3",
-    "rollup-plugin-uglify": "^6.0.4",
+    "rollup-plugin-terser": "^7.0.2",
```
Comment (jasonnutter) [inline lib/msal-browser/package.json:87]: @pkanher617 FYI

File: change/@azure-msal-node-extensions-2020-10-30-15-28-49-npm-audit.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```
Comment (jasonnutter) [inline change/@azure-msal-node-extensions-2020-10-30-15-28-49-npm-audit.json:2]:
```suggestion
  "type": "prerelease",
```

**Review by tnorling — APPROVED** [review]

**Review by jo-arroyo — APPROVED** [review]

File: .github/workflows/npm-audit.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+
+    - name: npm audit (root)
+      run: npm audit
+
+  audit-lib:
```
Comment (tnorling) [inline .github/workflows/npm-audit.yml:25]: Is this job needed? Won't lerna run the script against the repos in the first one?

Comment (jasonnutter) [reply .github/workflows/npm-audit.yml:25]: No, `npm audit` just runs against the root. I added `npm run audit` as a way to easily to this all locally, but doing them individually here to make it clear which ones fail (given when `npm run audit` is run, as soon as one fails, it stops running all of them).

Comment (tnorling) [reply .github/workflows/npm-audit.yml:25]: Oh gotcha - missed the nuance between `npm audit` and `npm run audit`

**Review by pkanher617 — APPROVED** [review]

**Pull request merged**